R/summarize_weighted_production.R

```diff
@@ -118,11 +118,11 @@ summarize_unweighted_production <- function(data, ..., with_targets = FALSE) {
   old_groups <- dplyr::groups(data)
 
   data <- data %>%
-    select(-c(
-      .data$id_loan,
-      .data$loan_size_credit_limit,
-      .data$loan_size_outstanding
-    )) %>%
+    select(-all_of(c(
+      "id_loan",
+      "loan_size_credit_limit",
+      "loan_size_outstanding"
+    ))) %>%
     distinct() %>%
     group_by(.data$sector_abcd, .data$technology, .data$year, ...)
```

Comment:
does `group_by()` not require the same treatment as `ungroup()`?

Reply:
Because they're not symmetric — `group_by()` has "mutate" semantics, `ungroup()` has "select" semantics.

Reply:
Thanks for the explanation!